WDAC/Recommended block rules: Add notes and link

illfated opened the pull request:
**Description:**

As agreed on in issue ticket #3642 (**Ambiguity**), this PR adds a note about creating two diff policies or merging them in a broad policy,
as well as adding a link to how to merge WDAC policies. 

As recommended by @Justinha , Windows Server 2019 is also added to the "Applies to" list at the top of this document.

Also, thanks to @Air-Git for requesting this clarification.

**Proposed changes:**
- add note about creating two diff policies or merging them in a broad policy
- add a link to how to merge WDAC policies
- add Windows Server 2019 to the "Applies to" list
- Cosmetic: add single compatibility MarkDown quote spacing to the existing Note.

**issue ticket closure or reference:**

Closes #3642

illfated commented:
Rich Text Diff view, for easy overview of the changes:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5112/files?short_path=540ffe4#diff-540ffe49bd32e95e8b2a5da5ba1a565e

Suggestions and recommendations are welcome, especially for correct wording in the link section heading at the end.

e0i commented:
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

JohanFreelancer9 commented on windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md:
```diff
@@ -1511,3 +1512,10 @@ Pick the correct version of each .dll for the Windows release you plan to suppor
 <br />
+
+> [!Note]
+> To create a policy that works on both version 1803 and 1809, you can create two diff policies, or merge them into one broader policy.
```
```suggestion
> To create a policy that works on both Windows 10, version 1803 and version 1809, you can create two different policies, or merge them into one broader policy.
```

JohanFreelancer9 commented on windows/security/threat-protection/windows-defender-application-control/microsoft-recommended-block-rules.md:
```diff
@@ -1511,3 +1512,10 @@ Pick the correct version of each .dll for the Windows release you plan to suppor
+> [!Note]
+> To create a policy that works on both version 1803 and 1809, you can create two diff policies, or merge them into one broader policy.
+
+## Further reading (How-To)
```
```suggestion
## More information
```

illfated commented:
Updated content, simplified in Rich Text Diff view:
https://github.com/MicrosoftDocs/windows-itpro-docs/pull/5112/files?short_path=540ffe4#diff-540ffe49bd32e95e8b2a5da5ba1a565e
